conference/react-js/src/containers/Conference-cards/Past/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import axios from 'axios';
+import baseUrl from '../../../constants/backend-url';
+
+class Past extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null,
+    };
+  }
+  componentWillMount() {
+    axios.get(`${baseUrl}/api/conference/past`)
+      .then((response) => {
+        const data = response.data;
+        this.setState({ data });
+      });
+  }
+  render() {
+    if (!this.state.data) {
+      return <div />;
+    }
+    const data = this.state.data;
+    const dataArray = Object.values(data);
+    return (
+      <div className="tabs-container">
+        {dataArray.map(element => (
+          <div className="tabs-container conference-card" key={element.id}>
+            <div className="conference-card-title">
+              <a
+                href="#/conference/{element.id}"
+                className="conference-card-title__link ng-binding"
+              >{element.title}</a>
+            </div>
+            <div className="conference-card-dates ng-binding">
+              <span className="conference-card-label">Dates:</span>
+              {element.start_date ? element.start_date : 'TBD'}
```

Comment:
You can use just: 
``` element.start_date || 'TBD' ```
If the first operand is false, You will get the second one.